test/fixtures/rules/std_rule_cases/L039.yml

```diff
@@ -112,3 +112,123 @@ test_redshift_at_time_zone:
   configs:
     core:
       dialect: redshift
+
+test_excess_space_without_align_alias:
+  fail_str: |
+        SELECT
+            a    AS first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo
+  fix_str: |
+    SELECT
+        a AS first_column,
+        b AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo
+  configs:
+    # This is the default config but we're being explicit
+    # here for testing.
+    layout:
+      type:
+        alias_expression:
+          spacing_before: single
+
+test_excess_space_with_align_alias:
+  # NOTE: The config here shouldn't move the table alias
+  fail_str: |
+        SELECT
+            a    AS first_column,
+            b      AS second_column,
+            (a + b) / 2 AS third_column
+        FROM foo   AS bar
+  fix_str: |
+    SELECT
+        a           AS first_column,
+        b           AS second_column,
+        (a + b) / 2 AS third_column
+    FROM foo AS bar
+  configs: &align_alias
+    layout:
+      type:
+        alias_expression:
```

Comment:
Where's the corresponding logic for aligning aliases? Not sure where to find it after it was moved from the other PR.

Reply:
as we discussed via slack (and below) the logic is in ` _determine_aligned_inline_spacing()`